diff --git a/node_modules/graphql/type/schema.js b/node_modules/graphql/type/schema.js
index 9eef4d0..555abfe 100644
--- a/node_modules/graphql/type/schema.js
+++ b/node_modules/graphql/type/schema.js
@@ -190,9 +190,9 @@ var GraphQLSchema = /*#__PURE__*/function () {
       var typeName = namedType.name;
       typeName || (0, _devAssert.default)(0, 'One of the provided types for building the Schema is missing a name.');
 
-      if (this._typeMap[typeName] !== undefined) {
-        throw new Error("Schema must contain uniquely named types but contains multiple types named \"".concat(typeName, "\"."));
-      }
+      // if (this._typeMap[typeName] !== undefined) {
+      //   throw new Error("Schema must contain uniquely named types but contains multiple types named \"".concat(typeName, "\"."));
+      // }
 
       this._typeMap[typeName] = namedType;
 
